service/src/intTest/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusTestUtils.java

```diff
@@ -31,10 +32,13 @@ public static EhrExtractStatus prepareEhrExtractStatus() {
             EhrStatusConstants.TO_ODS_CODE);
     }
 
-    private static List<EhrExtractStatus.GpcAccessDocument> prepareGpcAccessDocuments() {
-        return ImmutableList.of(EhrExtractStatus.GpcAccessDocument.builder()
-            .objectName(EhrStatusConstants.DOCUMENT_ID + ".json")
-            .build()
-        );
+    private static EhrExtractStatus.GpcAccessDocument prepareGpcAccessDocument() {
+        return EhrExtractStatus.GpcAccessDocument.builder()
+            .documents(List.of(
+                EhrExtractStatus.GpcAccessDocument.GpcDocument.builder()
```

Comment:
This is a lot of indentations. Could we move some of them to separate methods?

Reply:
I think this is fine for building a test fixture